ground/src/main/java/com/google/android/ground/model/geometry/ModelTypeTag.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.geometry
+
+/**
+ * Indicates the type of a geometric model object. Used to interpret other objects (e.g. map
+ * features) back into model objects.
+ */
+enum class ModelTypeTag {
```

Comment:
Nice! Should this class maybe live in the ui layer, since it's specific to UI interactions?

Maybe `FeatureType` would make sense here? Note that tag := id + type.

Reply:
done and done!